### ISSUE TITLE
Error in Step 3 of setting up your own RPC server

### ISSUE THREAD

**Issue by briwylde08** [stellar/stellar-docs#1051]
There is an error when following the guide to launch an RPC server. In this section: https://developers.stellar.org/docs/data/rpc/admin-guide#docker-image - in Step 3, the `--captive-core-use-db` flag gives an Error: unknown flag.

**Comment by ElliotFriend** [comment]
I'm pretty sure all we need to do here is remove the `--captive-core-use-db` flag from the example command. It was removed from the possible flags when captive core switched from postgres to bucketlistdb.

It would be worth running through the docker instance setup tests to confirm, though.

**Comment by anataliocs** [comment]
@ElliotFriend Yup yea I double-checked the available flags here:  https://github.com/stellar/stellar-rpc/blob/main/cmd/stellar-rpc/internal/config/options.go

And confirmed all the other flags are present but --captive-core-use-db has been removed.  

Can confirm via test config file as well:  https://github.com/stellar/stellar-rpc/blob/main/cmd/stellar-rpc/internal/config/test.soroban.rpc.config

I think this old config is still in https://github.com/stellar/quickstart as well.

Will put up a PR to update the docs but @ElliotFriend do you think I should update quickstart as well since that is commonly used?
